packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -75,6 +76,7 @@ function clientConfig(env) {
 		},
 
 		plugins: [
+			new CrittersPlugin(),
```

Comment:
this should be after `...RenderHTMLPlugin(env),` to resolve the error from https://github.com/GoogleChromeLabs/critters/issues/5